### PR TITLE
qa_crowbarsetup.sh: split long lines

### PR DESCRIPTION
**Pull request by aspiers**
Conventional wisdom across the industry is that it is a bad idea to exceed 80 column lines. This is visible in countless areas, from the Linux kernel coding style through to PEP8 and beyond.  Long lines definitely make code harder to read for some people, so even if you don't *personally* feel it increases readability for you, please still be considerate of other people.

It's not just a matter of taste either.  Some user interfaces do not automatically wrap lines, so in those cases long lines can cause significant usability issues.  For example, when viewing diffs on parts of github, the lack of line wrapping means that you have to use a horizontal scroll bar to see the whole line:

line-chopped

This is a major PITA.

In the case where there is an `if` statement with a long conditional, it means that the conditional gets broken across multiple lines.  In this case it's more readable if the `then` is on its own line, so you can clearly see where the conditional ends and the body begins.  This type of approach is advocated in

* http://wiki.bash-hackers.org/scripting/style

This requires ignoring a ridiculous bashate rule (`E011`) which complains if `then` is not on the same line as the `if` beginning a conditional.

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -491,10 +492,17 @@ function onadmin_prepare_sles_repos()
...
+        zypper se -s sles-release | \
+            grep -v -e "sp.up\s*$" -e "(System Packages)" | \
+            grep -q x86_64 \
+        || zypper ar \
+            http://$susedownload/install/SLP/SLES-${slesversion}-LATEST/x86_64/DVD1/ \
+            sles
 
         if ! $longdistance ; then
-            add_mount "" "clouddata.cloud.suse.de:/srv/nfs/suse-$suseversion/install" "${targetdir_install}"
+            add_mount "" \
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: Somehow I don't feel that this increases readability in the case of a single command with long args (and I am actually often using editor windows of 80 char width).

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: Conventional wisdom across the industry is that it is a bad idea to exceed 80 column lines.  This is visible in countless areas, from the Linux kernel coding style through to PEP8 and beyond.  Long lines definitely make code harder to read for me, so even if you don't feel it increases readability for you then please be considerate of other people.

Also don't forget that some user interfaces do not automatically wrap lines.  Which code in the below image do you think is more readable - the red or the green?

line-chopped

**Comment by nkrinner** [comment]
+1 I agree split lines add to readability

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: on the other hand, there is this
counter-example

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: The counter-example does not cancel out the fact that long lines cause breakage in some places, so it's not really relevant.  And even if github handled long lines perfectly, exceeding 80-column lines is still a) making life harder for other people and b) against the conventional wisdom of most of the industry.  You wouldn't try to submit a patch with long lines to an OpenStack project, so why do it here?

**Comment by aspiers** [comment]
Rebased, added some more line splits, and added a proper justification to the commit message.

**Comment by aspiers** [comment]
Rebased.  I believe I have justified this change and there are no unanswered objections to merging.  If that's not the case then please respond.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -479,26 +481,37 @@ function onadmin_prepare_sles_repos()
...
+            add_mount "$zypprepo" \
+                "clouddata.cloud.suse.de:/srv/nfs/repos/$repo" \
                 "$tftpboot_repos_dir/$repo"
         done
 
         # just as a fallback if nfs did not work
         if [ ! -e "$targetdir_install/media.1/" ]; then
             download_and_mount_sles "$tftpboot_suse_dir" "$targetdir_install"
+                http://$susedownload/install/SLES-$slesversion-$slesmilestone/$f \
+            || complain 70 "iso not found"
```
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: wrong rebase.

Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: Sorry :-(  This kind of thing can happen when PRs are open for a long time.  Fixed now.

**Pull request merged**